rpm/gnu-tar.spec

```diff
@@ -1,21 +1,22 @@
 %define _name tar
 Name:           gnu-%{_name}
-Version:        1.32
+Version:        1.35
 Release:        1
 Summary:        A GNU file archiving program
 License:        GPLv3+
 URL:            http://www.gnu.org/software/tar/
 Source:         %{name}-%{version}.tar.bz2
-Patch0:         tar-1.28-loneZeroWarning.patch
-Patch1:         tar-1.28-vfatTruncate.patch
-Patch2:         tar-1.29-wildcards.patch
+Patch0:         0001-Stop-issuing-lone-zero-block-warnings-downstram.patch
+Patch1:         0002-vfat-like-FS-sparse-still-downstream.patch
+Patch2:         0003-wildcard-defaults-downstram-compatibility.patch
+Patch3:         0004-Don-t-build-docs.patch
 BuildRequires:  gettext
 BuildRequires:  libacl-devel
 BuildRequires:  autoconf
 BuildRequires:  automake
 BuildRequires:  bison
-Provides:       %{_name} = 1.32+git2
-Obsoletes:      %{_name} < 1.32+git2
+Provides:       %{_name} = 1.35
+Obsoletes:      %{_name} < 1.35
```

Comment:
I think we should keep these as the old versions even if a bit confusing, otherwise there is a mismatch with tar provided by busybox which might make selecting correct version not work as it should if something requires "tar" and this would always get selected.